frontend/src/utils/get.ts

```diff
@@ -1,14 +1,12 @@
 import axios from 'axios';
 
-export const backendUrl = 'http://localhost:8080';
-
 export default function get<T>(
   route: string,
-  setState: (data: T[]) => void,
-  setLoading: ((loading: boolean) => void) | undefined
+  setState: (data: T) => void,
+  setLoading: ((loading: boolean) => void) | undefined = undefined
```

Comment:
Oh nice this is better. I noticed that most of the times we were using `get`, we were always passing in `undefined` anyways. Definitely will make my `landlord-cards` pr cleaner- there are so many GET requests in there.